ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -40,6 +40,12 @@ final case class Config[Extra](
     trackerRetentionPeriod: FiniteDuration,
     engineMode: EngineMode,
     enableAppendOnlySchema: Boolean, // TODO append-only: remove after removing support for the current (mutating) schema
+    indexerInputMappingParallelism: Int,
```

Comment:
Please group the new append-only schema related parameters into a separate case class for easier readability. This case class has already too many fields.

Reply:
:+1: